playground/backend/internal/fs_tool/fs.go

```diff
@@ -133,3 +142,37 @@ func (l *LifeCycle) GetExecutableName() (string, error) {
 func getFileName(pipelineId uuid.UUID, fileType string) string {
 	return fmt.Sprintf("%s.%s", pipelineId, fileType)
 }
+
+// setCorrectPath sets topLevelFolder as a current working folder.
+func setCorrectPath() error {
+	folder, err := getCurrentFolderName()
+	if err != nil {
+		return err
+	}
+
+	for !strings.EqualFold(folder, topLevelFolder) {
+		err = os.Chdir("..")
```

Comment:
can we avoid relative paths like ./ .. ../.. etc. ? If we don't use working dir we should avoid relative paths